code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2660,6 +2660,29 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		QDEL_NULL(prophet_trauma)
 	return ..()
 
+//a jacked up absinthe that causes hallucinations to the game master controller basically, used in smuggling objectives
+/datum/reagent/consumable/ethanol/ritual_wine
+	name = "Ritual Wine"
+	description = "The dangerous, potent, alcoholic component of ritual wine."
+	color = rgb(35, 231, 25)
+	boozepwr = 90 //enjoy near death intoxication
```

Comment:
90, all in all, isn't actually that much. Hooch has 300 iirc.

Reply:
thats bacchus blessing